src/main/java/com/ibm/usecases/compliance/queries/RequestComplianceCheckForScannedGitRepositoryQueryHandler.java

```diff
@@ -69,7 +69,7 @@ public RequestComplianceCheckForScannedGitRepositoryQueryHandler(
         final Collection<CryptographicAsset> cryptographicAssets =
                 compliancePreparationService.receiveCryptographicAssets(
                         this.readRepository,
-                        new GitUrl(requestComplianceCheckForScannedGitRepositoryQuery.gitUrl()),
+                        new ScanUrl(requestComplianceCheckForScannedGitRepositoryQuery.gitUrl()),
```

Comment:
For compliance either a `giturl` or `purl` should be the required input. A `scanUrl` should not be existing in the "compliance" domain

Reply:
I reactivated the old `GitUrl` class to represent the gitUrl in `ScanAggregate` which is passed in here.